src/main/java/core/basesyntax/custom/exceptions/AgeNotValidException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.custom.exceptions;
+
+public class AgeNotValidException extends RuntimeException {
```

Comment:
One Exception class is good enough.
```suggestion
public class RegistrationException extends RuntimeException {
```